assets/src/js/experiments/types.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Types for experiments.
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+export interface ExperimentsData {
+	[ key: string ]: boolean;
+}
+
+export interface ExperimentsObject {
+	[ key: string ]: boolean | ( ( name: string ) => boolean );
+	isEnabled: ( name: string ) => boolean;
+}
+
+export interface ACF {
+	experiments?: ExperimentsObject;
+	[ key: string ]: any;
+}
+
+declare global {
+	interface Window {
+		acf: ACF;
+		acfExperiments?: ExperimentsData; // Variable created by wp_localize_script in admin-experiments.php L150.
```

Comment:
Nit: I don't think exact line reference will age well. It could be also the case with the file name, so if you want to provide more context, you can list the PR instead.